consensus/service/README.md

```diff
@@ -142,16 +146,22 @@ An example run command is the below.
 
 >Note: The environment variables, `SGX_MODE`, `IAS_MODE`, `CONSENSUS_ENCLAVE_CSS` and `CONSENSUS_ENCLAVE_SIGNED` indicate important parameters to the SGX Enclave build. Please see [BUILD.md](./BUILD.md) for more details.
 
+>Note: Running in `IAS_MODE=DEV` runs a debug enclave.
+
 ```
-SGX_MODE=HW IAS_MODE=DEV CONSENSUS_ENCLAVE_CSS=consensus-enclave.css CONSENSUS_ENCLAVE_SIGNED=libconsensus-enclave.signed.so
+SGX_MODE=HW IAS_MODE=DEV \
+    CONSENSUS_ENCLAVE_CSS=$(pwd)/consensus-enclave.css \
+    CONSENSUS_ENCLAVE_SIGNED=$(pwd)/libconsensus-enclave.signed.so \
     cargo run --release -p consensus-service -- \
     --client-responder-id my_node.my_domain.com:443 \
-    --local-node-id node1.my_domain.com:8443 \
+    --peer-responder-id node1.my_domain.com:8443?consensus-msg-key=MCowBQYDK2VwAyEAdm6tv2Mqp1ZM6Uw0MvhaiMq85l5GPDCH8-BAiO6itAo= \
```

Comment:
I don't think the peer responder ID has the key in it...

Reply:
Will fix up!